storage/innobase/btr/btr0sea.cc

```diff
@@ -70,14 +65,11 @@ indexes. */
 
 /* We will allocate the latch from dynamic memory to get it to the
 same DRAM page as other hotspot semaphores */
-UNIV_INTERN rw_lock_t*		btr_search_latch_temp;
-
-/** padding to prevent other memory update hotspots from residing on
-the same memory cache line */
-UNIV_INTERN byte		btr_sea_pad2[64];
+UNIV_INTERN rw_lock_t*		btr_search_latch_temp
+					 MY_ALIGNED(CACHE_LINE_SIZE);
 
 /** The adaptive hash index */
-UNIV_INTERN btr_search_sys_t*	btr_search_sys;
+UNIV_INTERN btr_search_sys_t*	btr_search_sys MY_ALIGNED(CACHE_LINE_SIZE);
```

Comment:
This looks alright, however it may suffer from false sharing with subsequent variables.

Also I don't completely understand why btr_search_latch_temp and btr_search_sys pointers themselves have to be moved to separate cache lines. They're updated only during server startup. I guess actuall aim was to move btr_search_n_succ and btr_search_n_hash_fail to separate cache lines.